src/main/java/com/spikes2212/control/RioPIDLoop.java

```diff
@@ -117,14 +165,22 @@ public void disable() {
     }
 
 
-    public void enableContinuousInput(double minimumValue, double maximumValue) {
+    private void enableContinuousInput(double minimumValue, double maximumValue) {
```

Comment:
there is no need for this one-liner function. you can just run it in the constructor.